### ISSUE TITLE
gulp unable to minify js because of study_sorter.js

### ISSUE THREAD

**Issue by JonDevOps** [w3develops/w3develops#254]
I receive the following Error Message after running the "gulp js" or "gulp watch" commands:

GulpUglifyError: unable to minify JavaScript
Caused by: SyntaxError: Unexpected token: keyword «const»
File: /mnt/c/Users/offic/Desktop/workspace/w3develops/js/study_sorter.js
Line: 8
Col: 8

**Comment by waterlgndx** [comment]
Yeah, from what I've briefly read it breaks encapsulation so if we can resolve that we should be able to minify.  I'm not sure that const is supported during that process.  This needs refactored and reworked anyway.  It's possible that we could reincorporate js on the html page to include any constants, not sure yet.
